### PR TITLE
ADD: Check for special characters in a string

### PR DESCRIPTION
**Pull request by binkkatal**
ADD: Check for special characters in a string.

The Method ContainsSpecialChars, uses regex to validate a string.
The regex being used is `^[a-zA-Z0-9]*$`

### CODE REVIEW

**Comment by patil-ashutosh** [comment]
@binkkatal   Thank your for raising your pull request.

File: regex/rstring/string.go [hunk 1]
```diff
@@ -5,6 +5,10 @@ import (
 	"regexp"
 )
 
+var (
+	isStringAlphabetic = regexp.MustCompile(`^[a-zA-Z0-9]*$`).MatchString
```
Comment (patil-ashutosh) [inline regex/rstring/string.go:9]: @binkkatal please try to address these comments.

**Review by patil-ashutosh — CHANGES_REQUESTED** [review]
@binkkatal Please address comments. Feel free to ping if need any help

**Comment by binkkatal** [comment]
@patil-ashutosh fixed the lint issues and squashed the commits.
ready for re review

**Comment by binkkatal** [comment]
@patil-ashutosh  do i need to setup codecov token for this pr to be merged?

**Comment by patil-ashutosh** [comment]
> @patil-ashutosh do i need to setup codecov token for this pr to be merged?

@binkkatal No need to setup token , I fixed CodeCov issue so no need of token.
Please refresh branch once , issue will resolve

**Comment by binkkatal** [comment]
> > @patil-ashutosh do i need to setup codecov token for this pr to be merged?
> 
> @binkkatal No need to setup token , I fixed CodeCov issue so no need of token.
> Please refresh branch once , issue will resolve

@patil-ashutosh  i rebased with the latest master and the upload step breaks again.
can you have a look at it

**Pull request merged**